437 completed. Now we can select columns from a range dependent on pr…

akak1977 opened the pull request:
…ovider type.

akak1977 committed:
437 completed. Now we can select columns from a range dependent on provider type.

yecli commented on Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Localizations/en.VirtoCommerce.Export.json:
```diff
@@ -26,7 +26,7 @@
                     "exported-columns": "Select columns for export",
                     "all": "All",
-                    "provider-description": "Tabular providers (e.g. Csv provider) are unavailable for exported types which do not support tabular export.",
+                    "provider-description": "Available object types depends on provider. Some types does not support tabular export (CSV provider).",
```
Available exported types depend on selected provider. Some types do not support tabular export (CSV provider).

yecli reviewed (commented):
Please add tests for TabularConverter usage and TabularMetadata filtering.

yecli committed:
Merge branch 'export/dev' into export/437-Included-columns-for-tabular

akak1977 committed:
Review-based changes.
Check for TabularDataConverter and TabularMetadata appended in PriceCsvExport testchain.

akak1977 committed:
Merge branch 'export/437-Included-columns-for-tabular' of https://github.com/VirtoCommerce/vc-platform-core into export/437-Included-columns-for-tabular

akak1977 committed:
Default IncludedColumns initialization in ****ExportDataQuery does not need anymore.
